### DISCUSSION TITLE
An error has occurred while loading. (not accepting https://public-rest42.bullhornstaffing.com)

### DISCUSSION THREAD

**Discussion by ID-EricBorgen** [bullhorn/career-portal#459] [Q&A] [answered]
It looks like I'm getting 401 errors from the background calls on this, our new oscp page.

I feel like I'm missing something, because I can get results if I use https://rest42.bullhornstaffing.com instead of https://public-rest42.bullhornstaffing.com

This is the initial page from our wordpress plugin oscp
https://www.id-edd.com/wp-content/plugins/bullhorn-oscp/#/

If it helps any, this is our environment:
Current PHP version: 8.0.15
Current Wordpress version: 5.9

**Comment by charlesabarnes** [comment, marked as answer]
Hello, please contact bullhorn support to ensure proper setup

**Reply by ID-EricBorgen** [reply]
This isn't really an answer, but it let me know that the support department needed to make some change to our account for this to work.
